ambry-router/src/main/java/com/github/ambry/router/QuotaAwareOperationController.java

```diff
@@ -196,6 +196,8 @@ private void pollQuotaCompliantRequests(List<RequestInfo> requestsToSend,
             break;
           case REJECT:
             quotaAvailable = false;
+            logger.warn(
+                "Rejecting request for quota resource %s because of reject recommendation.", quotaResource.toString());
```

Comment:
nit: Probably need to be {} instead of %s for the argument.

Reply:
fixed.